### PR TITLE
Precision recall

### PR DESCRIPTION
**Pull request by ankurgupta-samsung**
Changes related to https://github.com/TVServices/emr_apps/issues/2

### CODE REVIEW

**Comment by ankurgupta-samsung** [comment]
Changes related to https://github.com/TVServices/emr_apps/issues/2

File: eventstream/src/main/java/com/samsung/tvservices/eventstream/main/spark/PrecisionRecallStatsGenerator.java [hunk 1]
```diff
@@ -0,0 +1,670 @@
...
+	}
+
+	public void startJob(String targetDate) {
+		SparkContextBuilder cb = new SparkContextBuilder();
+		JavaSparkContext sc = cb.readInputRecursively().build();
+		final HashMap<String, String> ESmap = getESConfigMap();
+		ESmap.put("es.mapping.id", "id");
+		// final String ENV =
+		// PropertyManager.getProperty(StreamAggregatedConstants.ENV_PROP);
+		final String ENV = "dev";
```
Comment (vinodtalati) [inline eventstream/src/main/java/com/samsung/tvservices/eventstream/main/spark/PrecisionRecallStatsGenerator.java]: Doesn't look like this is used/required.

File: eventstream/src/main/java/com/samsung/tvservices/eventstream/main/spark/PrecisionRecallStatsGenerator.java [hunk 2]
```diff
@@ -0,0 +1,670 @@
...
+			String watch_prefix = getS3DataSubFolderForEnv() + "/" + PropertyManager.getProperty(StreamAggregatedConstants.DATAFOLDER_CLICKEVENTS_PROP) + "/" + targetDate + "/";
+			System.out.println("Recommendation Prefix:" + recommendation_prefix);
+			System.out.println("Watch Prefix:" + watch_prefix);
+			final ListObjectsRequest listObjectRequest = new ListObjectsRequest().withBucketName(S3_BUCKET).withPrefix(recommendation_prefix).withDelimiter("/");
+			final List<String> objectListing = s3client.listObjects(listObjectRequest).getCommonPrefixes();
+			System.out.println("Object listing: " + objectListing);
+			for (String path : objectListing) {
+				String timestamp_folder = path.split("\\/")[path.split("\\/").length - 1];
+				System.out.println(timestamp_folder);
+				timestamp_folder = "13_30";
```
Comment (vinodtalati) [inline eventstream/src/main/java/com/samsung/tvservices/eventstream/main/spark/PrecisionRecallStatsGenerator.java]: remove hard coding. Assuming this was for testing purposes :)

File: eventstream/src/main/java/com/samsung/tvservices/eventstream/main/spark/PrecisionRecallStatsGenerator.java (hunk 1 above)
Comment (ankurgupta-samsung) [inline eventstream/src/main/java/com/samsung/tvservices/eventstream/main/spark/PrecisionRecallStatsGenerator.java]: We need this though for mapping: 

ESmap.put("es.mapping.id", "id");


Removed hard coded ENV value. It was just for testing

File: eventstream/src/main/java/com/samsung/tvservices/eventstream/main/spark/PrecisionRecallStatsGenerator.java (hunk 2 above)
Comment (ankurgupta-samsung) [inline eventstream/src/main/java/com/samsung/tvservices/eventstream/main/spark/PrecisionRecallStatsGenerator.java]: Yes, this was for testing. Removed it.

**Pull request merged**